src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -1232,37 +1242,46 @@ namespace winrt::TerminalApp::implementation
     {
         if (const auto terminalTab{ _GetFocusedTabImpl() })
         {
-            try
-            {
-                // TODO: GH#5047 - In the future, we should get the Profile of
-                // the focused pane, and use that to build a new instance of the
-                // settings so we can duplicate this tab/pane.
-                //
-                // Currently, if the profile doesn't exist anymore in our
-                // settings, we'll silently do nothing.
-                //
-                // In the future, it will be preferable to just duplicate the
-                // current control's settings, but we can't do that currently,
-                // because we won't be able to create a new instance of the
-                // connection without keeping an instance of the original Profile
-                // object around.
+            _DuplicateTab(*terminalTab);
+        }
+    }
 
-                const auto& profileGuid = terminalTab->GetFocusedProfile();
-                if (profileGuid.has_value())
+    // Method Description:
+    // - Duplicates specified tab
+    // Arguments:
+    // - tab: tab to duplicate
+    void TerminalPage::_DuplicateTab(const TerminalTab& tab)
```

Comment:
This is a nit, but mind renaming the function above to something like "_DuplicateCurrentTab"? I feel like somebody will get confused between `_DuplicateTab(tab)` and `_DuplicateTabViewItem()` at some point.

Reply:
Renamed into _DuplicateFocusedTab